packages/components/src/components/hive-footer/index.tsx

```diff
@@ -1,41 +1,44 @@
 import { ComponentProps, FC, ReactNode } from 'react';
-import { cn } from '../cn';
-import { siteOrigin } from '../constants';
-import { HiveCombinationMark } from '../logos';
-import { PRODUCTS } from '../products';
-import { IFooterExtendedProps, ILink } from '../types/components';
-import { Anchor } from './anchor';
+import { cn } from '../../cn';
+import { siteOrigin } from '../../constants';
+import { HiveCombinationMark } from '../../logos/index';
+import { PRODUCTS } from '../../products';
+import { ILink } from '../../types/components';
+import { Anchor } from '../anchor';
 import {
   CSAStarLevelOneIcon,
   DiscordIcon,
   GitHubIcon,
   LinkedInIcon,
   TwitterIcon,
   YouTubeIcon,
-} from './icons';
+} from '../icons/index';
+import { ContactUs } from './contact-us';
 
-export interface HiveFooterProps extends IFooterExtendedProps {
+export type HiveFooterProps = {
+  className?: string;
+  resources?: ILink[];
+  logo?: ReactNode;
+  href?: string;
```

Comment:
Makes sense. So instead of `resources: ILink[]` prop we could do sth like

```tsx
items={[
  {
    label: 'Products',
    items: [
      { children: 'Hive', href: 'https://the-guild.dev/graphql/hive' }
    ]
  },
  {
    children: 'OSS Friends',
    href: 'https://the-guild.dev/graphql/hive/oss-friends'
  }
]}
```